source/quakedef.h

```diff
@@ -42,6 +42,8 @@ extern int psp_system_model;
 #define	WINQUAKE_VERSION	0.996
 #define	LINUX_VERSION		1.30
 #define	X11_VERSION			1.10
+#define WIIGX_VERSION			0.09
```

Comment:
Not sure if its necessary to specify some gxQuake revision, we should always report the same engine version (I think it is `2.0` or something)